packages/adminrouter/extra/src/Makefile.common

```diff
@@ -56,26 +56,13 @@ devkit:
 	@if $$(docker images | grep mesosphere/$(DEVKIT_NAME) | grep -q full); then \
 		echo "Devkit image is present, not rebuilding it."; \
 	else \
-		echo "Building devkit image"; \
-		docker build \
-			--rm --force-rm \
-			-t mesosphere/$(DEVKIT_NAME):noresty \
-			-f ../../docker/Dockerfile \
-				../../docker/ && \
-		docker run \
-			$(DEVKIT_BASE_DOCKER_OPTS) \
-			mesosphere/$(DEVKIT_NAME):noresty \
-				/bin/bash -c "\$$OPENRESTY_COMPILE_SCRIPT" && \
-		docker commit $$(docker ps -a -q -f name=$(DEVKIT_NAME)) \
-			mesosphere/$(DEVKIT_NAME):full && \
-		docker rm $(DEVKIT_NAME) && \
-		docker rmi -f mesosphere/$(DEVKIT_NAME):noresty; \
+		$(MAKE) update-devkit \
 	fi
 
 .PHONY: update-devkit
 update-devkit: clean-devkit-container
 	@# Let's suppress ALL the output:
-	@docker rm $(DEVKIT_NAME) &> /dev/null || true
+	@docker rm -f $(DEVKIT_NAME) || true
```

Comment:
I think that you said that you would remove this, as it is covered by `clean-devkit-container`

Reply:
oh, missed that one :( Fixing.